.changes/unreleased/Features-20240318-033136.yaml

```
@@ -0,0 +1,6 @@
+kind: Features
+body: ' Add new workflow for internal patch releases'
```

If adapters wants these changelogs
```suggestion
body: 'Add new workflow for internal patch releases'
```

---

I've been tagging GH actions changes with `Skip Changelog` (since these aren't shipped to end users).

Do we want to do that here too?